auth0/resource_auth0_organization_member_test.go

```diff
@@ -0,0 +1,187 @@
+package auth0
+
+import (
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/auth0/go-auth0/management"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/auth0/terraform-provider-auth0/auth0/internal/template"
+)
+
+func init() {
+	resource.AddTestSweepers("auth0_organization_member", &resource.Sweeper{
```

Comment:
This seems to be exactly the same sweeper as in the organization file with the addition that after we delete the org we also clear the users. But if the org gets wiped there will be no more users to detach from an org, right?

We already have a test sweeper for org, users and roles separately. Those should be enough to clear up anything we might create within the org_member tests.

Reply:
Sweepers only run when invoked through the "-sweep" flag on the go test command, they don't actually run before running the organization member tests if we run it individually. To invoke them we have the `make test-sweep`  command. 

More from the docs:
```
// AddTestSweepers function adds a given name and Sweeper configuration
// pair to the internal sweeperFuncs map. Invoke this function to register a
// resource sweeper to be available for running when the -sweep flag is used
// with `go test`. Sweeper names must be unique to help ensure a given sweeper
// is only ran once per run.
```